docker/internal/tarfile/writer.go

```diff
@@ -340,21 +340,26 @@ type tarFI struct {
 func (t *tarFI) Name() string {
 	return t.path
 }
+
```

Comment:
Absolutely non-blocking: I’m not in love with these added lines. But then I can see argument that `tarFI` could be in a separate file instead…

Reply:
Perhaps this can be reformatted like this instead?

```go
func (t *tarFI) Name() string { return t.path }
  
func (t *tarFI) Size() int64 { return t.size }
  
func (t *tarFI) Mode() os.FileMode {
        if t.isSymlink {
                return os.ModeSymlink
        }
        return 0o444
}
  
func (t *tarFI) ModTime() time.Time { return time.Unix(0, 0) }
  
func (t *tarFI) IsDir() bool { return false }
  
func (t *tarFI) Sys() any { return nil }
```

Reply:
\*shrug\* That also works… I realize this is not worth that much discussion.